onnxruntime/core/providers/dml/dml_provider_factory.cc

```diff
@@ -55,6 +63,11 @@ std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_DML(ID
   return std::make_shared<onnxruntime::DMLProviderFactory>(dml_device, cmd_queue);
 }
 
+void DmlConfigureProviderFactoryDefaultRoundingMode(IExecutionProviderFactory* factory, AllocatorRoundingMode rounding_mode) {
```

Comment:
>DmlConfigureProviderFactoryDefaultRoundingMode [](start = 5, length = 46)

figure out which header this should live in.

Reply:
for a future time

---
In reply to: [370289892](https://github.com/microsoft/onnxruntime/pull/2891#discussion_r370289892) [](ancestors = 370289892)